### DISCUSSION TITLE
no icons

### DISCUSSION THREAD

**Discussion by Hello-LuisWu** [sxyazi/yazi#1399] [1. Q&A]
None of the configured ICONS has taken effect  ????  [ theme config file ](https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/theme.toml)

**Comment by lpnh** [comment]
This is the default configuration file for Yazi 0.3.0 (latest release). You don't need to add anything to your own configuration for it to take effect. If you are customizing it, just make sure you are running Yazi 0.3.0. You can check the version with the `yazi --version` command.

**Reply by Hello-LuisWu** [reply]
thanks

**Comment by uncenter** [comment]
I'm guessing you have an outdated theme configuration file. In v0.3.0, the options changed so your old configuration won't work. You should copy the new options from the new preset file.
